shared/params/loader.go

```diff
@@ -50,7 +50,8 @@ func LoadChainConfigFile(chainConfigFileName string) {
 	OverrideBeaconConfig(conf)
 }
 
-func replaceHexStringWithYAMLFormat(line string) []string {
+// ReplaceHexStringWithYAMLFormat will replace hex strings that the yaml parser will understand.
+func ReplaceHexStringWithYAMLFormat(line string) []string {
```

Comment:
Looks like this is exported only to satisfy tests which have been moved to `params_test` package. Is this OK?

Reply:
Yes